src/components/specs/Container.spec.js

```diff
@@ -0,0 +1,13 @@
+import { create } from 'react-test-renderer';
+import Container from '../Container';
+
+jest.mock('react-router-dom', () => ({ Redirect: 'Redirect', Route: 'Route', Switch: 'Switch', useLocation: () => ({ pathname: '/' }) }));
+jest.mock('../../pages', () => ({ Locate: 'Locate', Details: 'Details', Home: 'Home', Search: 'Search' }));
+
+
+describe('Container', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/punny-weather/pullRequest?prid=7693782)
```suggestion
describe("Container", () => {
```